bindings/nodejs/examples/how_tos/wallet/create-wallet.ts

```diff
@@ -44,18 +44,7 @@ async function run() {
         // The mnemonic can't be retrieved from the Stronghold file, so make a backup in a secure place!
         await secretManager.storeMnemonic(process.env.MNEMONIC as string);
 
-        const walletAddress = await secretManager.generateEd25519Addresses({
```

Comment:
wait, why remove?

Reply:
I am not sure the default `bipPath` fields are the same as the one that were used for the address though, idk if it changes anything for the example

Reply:
yeah that's what I mean, the values are different. but 🤷‍♀️  if that's not a problem then fine

Reply:
@Alex6323 is it not a problem?

Reply:
There is no default, but it's provided (another one would also not be a problem)
```
            bipPath: {
                coinType: CoinType.IOTA,
            },
```

Reply:
What about accountIndex? bech32Hrp? Irrelevant for this test? And for all tests where it's removed?